enh(build) Add Subresource Integrity digests for CDN builds (#2892)
highlightjs/highlight.js: tools/templates/DIGESTS.md

## Subresource Integrity

If you are loading Highlight.js via CDN you may wish to use [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) to guarantee that you are using a legimitate build of the library.

> To do this you simply need to add a `digest` attribute for each JavaScript file you download via CDN. These digests are used by the browser to confirm the files downloade have not been modified.

Reviewer: ```suggestion
To do this you simply need to add the `integrity` attribute for each JavaScript file you download via CDN. These digests are used by the browser to confirm the files downloaded have not been modified.
```

Did you mean the `integrity`?

```html
<script
  src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/<!-- $VERSION -->/highlight.min.js"

>   digest="<!-- $MIN_JS_DIGEST -->"></script>

Reviewer: ```suggestion
  integrity="<!-- $MIN_JS_DIGEST -->"></script>
```

Author: LOL. Oops. Not sure how I made that mistake.

<!-- including any other grammars you might need to load -->
<script
  src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/<!-- $VERSION -->/languages/go.min.js"

>   digest="<!-- $GO_SHA -->"></script>

Reviewer: ```suggestion
  integrity="<!-- $GO_SHA -->"></script>
```
